src/tplink-safeloader.c

```diff
@@ -9,7 +9,7 @@
    tplink-safeloader
 
    Image generation tool for the TP-LINK SafeLoader as seen on
-   TP-LINK Pharos devices (CPE210/220/510/520)
+   TP-LINK Pharos devices (CPE210/220/510/520/710)
```

Comment:
This was (probably) the original set of supported devices, but this list is no longer updated and would be far too long if we added all supported devices. Please drop this line.

Reply:
Removed the change